app/styles/routes/groups.scss

```diff
@@ -0,0 +1,4 @@
+.groups-avatar {
+  max-height: 75vh;
+  object-fit: contain !important;
```

Comment:
Is the !important needed here? Haven't tried it but I think this object-fit should overwrite the .card-img-rounded object-fit without !important, since this style file is imported later.

Reply:
I have also not tried it without the !important so I would not know